### ISSUE TITLE
Css class is not added to the dialog wrapper div with bod-block-popup-wrap active

### ISSUE THREAD

**Issue by spencer-j** [merbird/modal-block#3]
Hey merbird,

thank you for the nice needed plugin.
seems that the custom css class, addable trough gutenberg beneath the Style dropdown isn't added in frontend rendering.
In the backend code editor view it is shown. 
`<!-- wp:bod/modal-block {"showOn":"selector","triggerSelector":"triggerclass","className":"abc__modal"} -->`

Also in frontend there is a missmatch at aria-labelledby bodModalAriaTitle1 but the Id of the title is "bodModalAriaTitle2" also aria-describedby="bodModalAriaContent1"  id is "bodModalAriaContent2"

greets spencer

**Comment by merbird** [comment]
Hi 

That second problem with the aria should now be fixed I will take a look at your first issue

**Comment by merbird** [comment]
OK so I took a look at the first problem and it does add the class. The class is added to the main block wrapper. The way the modal works is that when the modal is activated, the dialog (the actual modal display) is moved outside the main code and placed just before the </body> tag. This ensures nothing else in the code impacts the modal. Unfortunately, this also means it's not inside the block container anymore, so the class could not be used to apply custom CSS. So I added the custom class to the dialog as well as the main block wrapper.
